scripts/st2bootstrap-deb.sh

```diff
@@ -350,7 +350,11 @@ generate_symmetric_crypto_key_for_datastore() {
   # set path to the key file in the config
   sudo crudini --set /etc/st2/st2.conf keyvalue encryption_key_path ${DATASTORE_ENCRYPTION_KEY_PATH}
 
-  sudo st2ctl restart-component st2api
+  # NOTE: We need to restart all the affected services so they pick the key and load it in memory
+  # NOTE: Only st2api, st2sensorcontainer, st2workflowengine and st2actionrunner work with
+  # encrypted datastore values, but we restart all the services to be on the safe side and for
+  # future proofing reasons.
+  sudo st2ctl restart
```

Comment:
I'd like to still avoid restarting everything

Also see for example where we follow this granular strategy, instead of restart-all:
https://github.com/StackStorm/st2-packages/blob/9249e5cdff0d6d8ead2d6921e7bc5028e35c9c61/scripts/st2bootstrap-deb.sh#L579-L585

Reply:
@armab Fine, I'll revert it to restarting just the affected services for now. It just means we need to be careful in the future or it will bite us again.

I honestly don't think 10-15s slow down is a big deal, but fine.

Reply:
Changing config without reloading it in every component is very dangerous. None of us could be able to tell from the top of our head whether any particular line of config used in one component or a several of them.

Reply:
Also, when adding new component or a feature to existing component you don't usually think about all the cases this component needs to be restarted during install.

Reply:
Then we have to keep in mind about this in future,
because in Docker we're giving granular access to specific services that _really_ need that access, not "free-for-all".

Ex: pack content and virtualenvs only for `st2actionrunner`, `st2sensorcontainers` and for register-content one-time task, (now `st2workflowengine` may need it as well ?).

Have to figure out the borders for each service.